blockschaltbilder/boilerplate.py

```diff
@@ -216,7 +216,12 @@ def convert_to_tikz(paths):
     for p in paths:
         if os.path.isdir(p):
             for file in _find_bsb_files(p):
-                _convert_single_file(file)
+                try:
+                    _convert_single_file(file)
+                except ValueError as e:
+                    print("ValueError in {0}: ".format(file), e)
```

Comment:
Could you please use `"ValueError in {:s}:"` instead?
* `{:s}` just for the sake of consistency, I used it this way
* remove the whitespace since `print()` puts one between args anyway